website/docs/r/region.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_region"
+sidebar_current: "docs-vcfa-resource-region"
+description: |-
+  Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+---
+
+# vcfa\_region
+
+Provides a resource to manage Regions in VMware Cloud Foundation Automation.
+
+~> Only `System Administrator` can create this resource.
+
+## Example Usage
+
+```hcl
+data "vcfa_supervisor" "one" {
+  name = "first-supervisor"
+
+  depends_on = [vcfa_vcenter.one]
```

Comment:
Should be a reference to vCenter ID:

`vcenter_id = vcd_vcenter.t.id`

Reply:
True, thanks for spotting!